src/gigs-board/pages/Create.jsx

```diff
@@ -0,0 +1,526 @@
+/* INCLUDE: "common.jsx" */
```

Comment:
Just so everyone is on the same page, it seems we cannot merge this PR without #122.